analyzers/tests/SonarAnalyzer.Test/Helpers/KnownAssemblyTest.cs

```diff
@@ -255,103 +255,115 @@ public void Combinator_StartsWith_Start_Or_EndsWith_End_2(string name, bool expe
     }
 
     [TestMethod]
-    public void XUnitAssert_2_4()
```

Comment:
Suggestion to reviewer: Read this file starting from the bottom, I added a helper method to avoid duplication and everything seems shifted, but it's just house-cleaning.